flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/catalog/PostgresCatalogTestBase.java

```diff
@@ -138,78 +143,68 @@ public static TestTable getSimpleTable() {
-				.field("int_arr", DataTypes.ARRAY(DataTypes.INT()))
 				.field("bytea", DataTypes.BYTES())
-				.field("bytea_arr", DataTypes.ARRAY(DataTypes.BYTES()))
 				.field("short", DataTypes.SMALLINT())
-				.field("short_arr", DataTypes.ARRAY(DataTypes.SMALLINT()))
 				.field("long", DataTypes.BIGINT())
-				.field("long_arr", DataTypes.ARRAY(DataTypes.BIGINT()))
 				.field("real", DataTypes.FLOAT())
-				.field("real_arr", DataTypes.ARRAY(DataTypes.FLOAT()))
 				.field("double_precision", DataTypes.DOUBLE())
-				.field("double_precision_arr", DataTypes.ARRAY(DataTypes.DOUBLE()))
 				.field("numeric", DataTypes.DECIMAL(10, 5))
-				.field("numeric_arr", DataTypes.ARRAY(DataTypes.DECIMAL(10, 5)))
 				.field("boolean", DataTypes.BOOLEAN())
-				.field("boolean_arr", DataTypes.ARRAY(DataTypes.BOOLEAN()))
 				.field("text", DataTypes.STRING())
-				.field("text_arr", DataTypes.ARRAY(DataTypes.STRING()))
 				.field("char", DataTypes.CHAR(1))
-				.field("char_arr", DataTypes.ARRAY(DataTypes.CHAR(1)))
 				.field("character", DataTypes.CHAR(3))
-				.field("character_arr", DataTypes.ARRAY(DataTypes.CHAR(3)))
 				.field("character_varying", DataTypes.VARCHAR(20))
-				.field("character_varying_arr", DataTypes.ARRAY(DataTypes.VARCHAR(20)))
 				.field("timestamp", DataTypes.TIMESTAMP(5))
-				.field("timestamp_arr", DataTypes.ARRAY(DataTypes.TIMESTAMP(5)))
-				.field("timestamptz", DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(4))
-				.field("timestamptz_arr", DataTypes.ARRAY(DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(4)))
+//				.field("timestamptz", DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(4))
 				.field("date", DataTypes.DATE())
-				.field("date_arr", DataTypes.ARRAY(DataTypes.DATE()))
-				.field("time", DataTypes.TIME(3))
-				.field("time_arr", DataTypes.ARRAY(DataTypes.TIME(3)))
+//				.field("time", DataTypes.TIME())
 				.build(),
 			"int integer, " +
-				"int_arr integer[], " +
 				"bytea bytea, " +
-				"bytea_arr bytea[], " +
 				"short smallint, " +
-				"short_arr smallint[], " +
 				"long bigint, " +
-				"long_arr bigint[], " +
 				"real real, " +
-				"real_arr real[], " +
 				"double_precision double precision, " +
-				"double_precision_arr double precision[], " +
 				"numeric numeric(10, 5), " +
-				"numeric_arr numeric(10, 5)[], " +
 				"boolean boolean, " +
-				"boolean_arr boolean[], " +
 				"text text, " +
-				"text_arr text[], " +
 				"char char, " +
-				"char_arr char[], " +
 				"character character(3), " +
-				"character_arr character(3)[], " +
 				"character_varying character varying(20), " +
-				"character_varying_arr character varying(20)[], " +
 				"timestamp timestamp(5), " +
-				"timestamp_arr timestamp(5)[], " +
-				"timestamptz timestamptz(4), " +
-				"timestamptz_arr timestamptz(4)[], " +
-				"date date, " +
-				"date_arr date[], " +
-				"time time(3), " +
-				"time_arr time(3)[]"
+//				"timestamptz timestamptz(4), " +
+				"date date",
+//				"time time",
```

Comment:
We do support `TIME(0)`.  Does it work to declare it as `time(0)`?

Reply:
it works. I'll make the change